docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM astral/uv:python3.12-trixie-slim AS ndif_base
```

Comment:
Just a clean Debian image. But it comes with Python 3.12 and uv installed. Allows us to skip building a Python venv if we don't need it (and just use system Python).